### PR TITLE
Array-chunk.md has been added and is up for review

### LINKED ISSUE #2875: [Term Entry] PHP Arrays array_chunk()
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `array_chunk()` function.

The entry should go in a new file in `docs/content/php/concepts/arrays/terms/array-chunk/array-chunk.md` (Note the dashes rather than underscores) 

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```php ` code block. 
- A runnable codebyte example in a ` ```codebyte/php` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by MellowZippy1**
### Description

Adds a term entry of PHP Arrays array_chunk(). I have done this due to the Git & GitHub course on Codecademy. I'm doing this course because the Computer Science career path advised me to.
Closes #2875 

### Type of Change

- Adding a new entry

### Checklist

- [ x ] All writings are my own.
- [ x ] My entry follows the Codecademy Docs style guide.
- [ x ] My changes generate no new warnings.
- [ x ] I have performed a self-review of my own writing and code.
- [  x] I have checked my entry and corrected any misspellings.
- [ x] I have made corresponding changes to the documentation if needed.
- [ x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by SSwiniarski** [comment]
@MellowZippy1 You need to sign the [Contributor License Agreement](https://cla-assistant.io/Codecademy/docs?pullRequest=2945). You also need to make sure both emails you're using for this PR are attached to your GitHub account.

File: content/php/concepts/arrays/terms/array-chunk/array-chunk.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+The **`array_chunk()`** method splits an array into chunks and returns a multidimensional array, each array containing `length` elements.
+
+## Syntax
+
+```pseudo
+array_chunk(array $array, int $length, bool $preserve_keys = false)
+```
+
+The `array_chunk()` function has two required parameter and one optional parameter.
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-chunk/array-chunk.md:22]:
```suggestion
The `array_chunk()` function has two required parameters and one optional parameter.
```

File: content/php/concepts/arrays/terms/array-chunk/array-chunk.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+- `$array`: Specifies the input `array`.
+- `$length`: Specifies that amount of elements in a chunk.
+- `$preserve_keys`: When set to **true** keys will be preserved. The default is **false** which will reindex each chunk.
+
+The `array_chunk()` function returns a multidimensional array, with each dimension containing **length** elements.
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-chunk/array-chunk.md:28]:
```suggestion
The `array_chunk()` function returns a multidimensional array, with each dimension containing `$length` elements.
```

File: content/php/concepts/arrays/terms/array-chunk/array-chunk.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+```pseudo
+array_chunk(array $array, int $length, bool $preserve_keys = false)
+```
+
+The `array_chunk()` function has two required parameter and one optional parameter.
+
+- `$array`: Specifies the input `array`.
+- `$length`: Specifies that amount of elements in a chunk.
+- `$preserve_keys`: When set to **true** keys will be preserved. The default is **false** which will reindex each chunk.
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-chunk/array-chunk.md:26]:
```suggestion
- `$preserve_keys`: When set to `true` keys will be preserved. The default is `false` which will reindex each chunk.
```

File: content/php/concepts/arrays/terms/array-chunk/array-chunk.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+            [15] => Peter
+            [16] => Quinn
+            [17] => Ruby
+            [18] => Sophia
+        )
+
+)
+```
+
+> **Note:** The last chunk may contain less than **length** elements. Due to $preserve_keys being set to **true** the indexes in the chunks do not reindex.
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-chunk/array-chunk.md:91]:
```suggestion
> **Note:** The last chunk may contain less than `$length` elements. Due to `$preserve_keys` being set to `true` the indexes in the chunks do not reindex.
```

File: content/php/concepts/arrays/terms/array-chunk/array-chunk.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+
+## Codebyte Example
+
+This example is runnable and uses the `array_chunk()` function:
+
+> **Note:** The example used a `foreach` loop to cycle through the elements with each chunk. `implode()` is used to join the names of students in each chunk to create a readable format for output.
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-chunk/array-chunk.md:97]:
```suggestion
> **Note:** The example uses a `foreach` loop to cycle through the elements with each chunk. `implode()` is used to join the names of students in each chunk to create a readable format for output.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@MellowZippy1 I did a first review and left a few comments.

**Comment by SSwiniarski** [comment]
@MellowZippy1 In the interest of moving this along, I committed my suggestions so this can go on to a second review.

**Review by SSwiniarski — APPROVED** [review]

File: content/php/concepts/arrays/terms/array-chunk/array-chunk.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
+---
+Title: 'array_chunk()'
+Desciprtion: 'Splits an array into chunks'
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-chunk/array-chunk.md:3]:
```suggestion
Description: 'Splits an array into a number of specified chunks.'
```

Comment (caupolicandiaz) [reply content/php/concepts/arrays/terms/array-chunk/array-chunk.md:3]: the spacing within the metadata section looks incorrect. Please look at one of the templates or an existing doc for reference

File: content/php/concepts/arrays/terms/array-chunk/array-chunk.md [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+    - 'learn-php'
+    - 'paths/computer-science'
+---
+
+The **`array_chunk()`** method splits an array into chunks and returns a multidimensional array, each array containing `length` elements.
+
+## Syntax
+
+```pseudo
+array_chunk(array $array, int $length, bool $preserve_keys = false)
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-chunk/array-chunk.md:19]:
```suggestion
array_chunk($array, $length, $preserve_keys)
```

File: content/php/concepts/arrays/terms/array-chunk/array-chunk.md [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+```
+
+The `array_chunk()` function has two required parameters and one optional parameter.
+
+- `$array`: Specifies the input `array`.
+- `$length`: Specifies that amount of elements in a chunk.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-chunk/array-chunk.md:25]:
```suggestion
- `$length`: An integer that specifies that amount of elements in a chunk.
```

File: content/php/concepts/arrays/terms/array-chunk/array-chunk.md [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+            [15] => Peter
+            [16] => Quinn
+            [17] => Ruby
+            [18] => Sophia
+        )
+
+)
+```
+
+> **Note:** The last chunk may contain less than `$length` elements. Due to `$preserve_keys` being set to `true` the indexes in the chunks do not reindex.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-chunk/array-chunk.md:91]:
```suggestion
> **Note:** The last chunk may contain less than `$length` elements.
```

File: content/php/concepts/arrays/terms/array-chunk/array-chunk.md [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+
+- `$array`: Specifies the input `array`.
+- `$length`: Specifies that amount of elements in a chunk.
+- `$preserve_keys`: When set to `true` keys will be preserved. The default is `false` which will reindex each chunk.
+
+The `array_chunk()` function returns a multidimensional array, with each dimension containing `$length` elements.
+
+## Example
+
+The following example uses the `array_chunk()` function to split an array of students into chunks of 5.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-chunk/array-chunk.md:32]:
```suggestion
The following example uses the `array_chunk()` function to split an array of students into chunks of 5. Additionally, the `$preserve_keys` parameter is set as `true`, which maintains the existing indexes.
```

File: content/php/concepts/arrays/terms/array-chunk/array-chunk.md [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+)
+```
+
+> **Note:** The last chunk may contain less than `$length` elements. Due to `$preserve_keys` being set to `true` the indexes in the chunks do not reindex.
+
+## Codebyte Example
+
+This example is runnable and uses the `array_chunk()` function:
+
+> **Note:** The example uses a `foreach` loop to cycle through the elements with each chunk. `implode()` is used to join the names of students in each chunk to create a readable format for output.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-chunk/array-chunk.md:97]:
```suggestion
> **Note:** The example uses a `foreach` loop to cycle through the elements with each chunk, and `implode()` is used to join the names of students in each chunk to create a readable format.
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@MellowZippy1 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Review by caupolicandiaz — APPROVED** [review]
@MellowZippy1 @SSwiniarski Since there hasn't been any activity on this entry, I've gone ahead and committed the suggestions I proposed. This is ready to merge

**Pull request merged**